.speakeasy/workflow.yaml

```diff
@@ -1,25 +1,25 @@
 workflowVersion: 1.0.0
 speakeasyVersion: latest
 sources:
-    speakeasy-OAS:
-        inputs:
-            - location: https://www.speakeasy.com/openapi.yaml
-        overlays:
-            - location: overlay.yaml
-        transformations:
-            - filterOperations:
-                operations: generateCodeSamplePreview, generateCodeSamplePreviewAsync, getCodeSamplePreviewAsync
-                include: true
-        registry:
-            location: registry.speakeasyapi.dev/speakeasy-self/speakeasy-public/speakeasy-oas
+  speakeasy-OAS:
+    inputs:
+      - location: https://www.speakeasy.com/openapi.yaml
+    overlays:
+      - location: overlay.yaml
```

Comment:
Dont think we need this overlay anymore

Reply:
Think your changes can just update the OAS directly

Reply:
I think I don't understand how this works - wouldn't we want to keep the original OAS + overlay so that it's updated on each generation?